### DISCUSSION TITLE
Freshdesk Shout out

### DISCUSSION THREAD

**Discussion by tanjaamplo** [Amplo-GmbH/Platform-Documentation#8] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/91560861/135421189-a4f86a6d-5774-446c-b4bf-4af3a3e72920.png)
Would it make sense to partner up with Freshdesk?

**Comment by nielsuit227** [comment]
Let's see if we qualify, they do indeed have a whole [network](https://www.freshworks.com/company/partners/).
Want to pick that up @tanjaamplo ?

**Comment by tanjaamplo** [comment, marked as answer]
Yes :)
